Makefile.in

```diff
@@ -195,11 +195,16 @@ ifeq ($(shell uname), Darwin)
 endif
 
 #
-# All translation message catalogs
+# All translation message catalogs, filter files based on LINGUAS.
 #
 TRANSLATIONS_SRC := $(wildcard po/*.po)
-ifdef HAVE_GETTEXT
+ifeq ($(HAVE_GETTEXT), 1)
```

Comment:
it seems if you build with --disable-gettext configure script will leave HAVE_GETTEXT empty.  that's why ```ifdef``` worked in the past.
but ifeq is more explicit and maybe more correct.

simple test:

add this target to Makefile.in
```make
echogettext:
        @echo "HAVE_GETTEXT    $(HAVE_GETTEXT)"
```


```fish
./configure --without-gettext
....
~ make echogettext
  FISH_BUILD_VERSION = ''
  FISH_BUILD_VERSION = '2.5.0-211-ge8eb436-dirty'
  HAVE_GETTEXT
```


repeat with


```fish
./configure --with-gettext
...

~ make echogettext
  FISH_BUILD_VERSION = '2.5.0-211-ge8eb436-dirty'
  HAVE_GETTEXT    1
```